src/Bigtable/Connection/Grpc.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Connection;
+
+use Google\ApiCore\Call;
+use Google\ApiCore\Serializer;
+use Google\Cloud\Core\GrpcRequestWrapper;
+use Google\Cloud\Core\GrpcTrait;
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient;
+use Google\Cloud\Bigtable\Admin\V2\Gapic\BigtableTableAdminGapicClient;
+use Google\Cloud\Bigtable\V2\Gapic\BigtableGapicClient;
+use Google\Cloud\Bigtable\Admin\V2\Table;
+use Google\Protobuf;
+use Google\Protobuf\FieldMask;
+use Google\Protobuf\GPBEmpty;
+use Google\Protobuf\ListValue;
+use Google\Protobuf\Struct;
+use Google\Protobuf\Value;
+use Grpc\UnaryCall;
+/**
+ * Connection to Cloud Bigtable over GRPC
+ */
+class Grpc implements ConnectionInterface
+{
+    use GrpcTrait;
+
+    /**
+     * @var Serializer
+     */
+    private $serializer;
+
+    /**
+     * @var InstanceAdminClient
```

Comment:
This, and the others typehints below should be the actual classname, e.g. `BigtableInstanceAdminClient`.